api/external/applications/applications.proto

```diff
@@ -87,6 +151,13 @@ service ApplicationsService {
     option (chef.automate.api.iam.policy).action = "applications:serviceGroups:delete";
   };
 
+  /*
+  Show 'Disconnected Services' configuration
+
+  Returns the configuration for the task that marks services as disconnected after
+  'threshold'. Threshold is a string that follows elasticsearch's date math expressions.
+  This job cannot be disabled, and therefor no information about running or not is provided.
+  */
   rpc GetDisconnectedServicesConfig(GetDisconnectedServicesConfigReq) returns (PeriodicMandatoryJobConfig) {
```

Comment:
Shouldn't `retention` be its own service?  https://github.com/chef/automate/tree/master/components/automate-gateway/api/compliance

Reply:
Can you elaborate on what you mean? Why can't we group actions under headings?

Reply:
Nevermind, I suspect this is related to https://github.com/chef/automate/pull/2286#issuecomment-563480699